quinn-proto/src/connection/mod.rs

```diff
@@ -197,7 +197,8 @@ pub struct Connection {
     //
     // Queued non-retransmittable 1-RTT data
     //
-    path_response: Option<PathResponse>,
+    /// Responses to PATH_CHALLENGE frames, and the addresses to send them to
+    path_responses: PathResponses,
```

Comment:
It would be nice to extract the abstraction in one commit, then update it to do the new stuff in another.

Reply:
Done.